Lib/test/test_zoneinfo/test_zoneinfo.py

```diff
@@ -1507,6 +1530,26 @@ def test_clear_cache_two_keys(self):
         self.assertIsNot(dub0, dub1)
         self.assertIs(tok0, tok1)
 
+    def test_clear_cache_refleak(self):
```

Comment:
Apparently I wrote this function 4 years ago, but coming back to it now it is opaque as heck, so we should probably describe what this test is about.

```suggestion
    def test_clear_cache_refleak(self):
        # bpo-43132: The portion of the C implementation of ZoneInfo that
        # ejects specific keys from the strong cache (i.e. the LRU cache) was
        # leaking references if comparing to the key object failed; because the
        # strong cache is not currently implemented for subclasses, this test
        # only really applies to the C implementation of the ZoneInfo class.
```